lib/bundler/cli.rb

```diff
@@ -199,6 +199,8 @@ def install
       "Force downloading every gem."
     method_option "ruby", :type => :boolean, :banner =>
       "Update ruby specified in Gemfile.lock"
+    method_option "bundler", :type => :string, :lazy_default => "> 0.a", :banner =>
```

Comment:
Not sure about defaulting people into a prerelease without them asking for it. Do you feel like that's okay to do?